### PR TITLE
16 docs

### PR DESCRIPTION
**Pull request by Ohjeah**

### CODE REVIEW

File: pysindy/feature_library/custom_library.py [hunk 1]
```diff
@@ -77,59 +73,53 @@ def get_feature_names(self, input_features=None):
...
-                    ),
+                    lambda i: (lambda *x: "f" + str(i) + "(" + ",".join(x) + ")"),
                     range(len(self.functions)),
                 )
             )
         return self
 
     def transform(self, X):
-        """Transform data to custom features
+        """Transform data to custom featuresö
```
Comment (briandesilva) [inline pysindy/feature_library/custom_library.py]: Was this intentional?

Comment (Ohjeah) [reply pysindy/feature_library/custom_library.py]: No, that's a typo.

Comment (briandesilva) [inline pysindy/feature_library/custom_library.py]:
```suggestion
        """Transform data to custom features
```

**Comment by briandesilva** [comment]
Looks good to me!

**Pull request merged**